tests/check/test-utils.c

```diff
@@ -119,10 +119,11 @@ test_variant_to_nevra(void)
   GError *error = NULL;
 
   g_autoptr(GFile) repo_path = g_file_new_for_path ("repo");
-  g_autoptr(OstreeRepo) repo = ostree_repo_new (repo_path);
-  ret = ostree_repo_create (repo, OSTREE_REPO_MODE_BARE_USER, NULL, &error);
+  g_autoptr(OstreeRepo) repo =
+    ostree_repo_create_at (AT_FDCWD, "repo", OSTREE_REPO_MODE_BARE_USER,
+                           NULL, NULL, &error);
+  g_assert (repo);
```

Comment:
Nit: can we move this assert underneath the `g_assert_no_error`? That way we'll actually see the error if this somehow fails.